Pod/Classes/OAStackView.m

```diff
@@ -16,6 +16,7 @@
 
 @interface OAStackView ()
 @property(nonatomic, copy) NSArray *arrangedSubviews;
```

Comment:
yes its used in line 76 but the backing property and its Ivar are not needed. I only removed the property from the .m file, the .h file still has the readonly one.